ocs_ci/framework/conf/default_config.yaml

```diff
@@ -41,11 +41,11 @@ REPORTING:
 # --cluster-conf file.yaml data you will pass to the pytest.
 ENV_DATA:
   cluster_name: null  # will be changed in ocscilib plugin
-  cluster_namespace: 'rook-ceph'
+  cluster_namespace: 'openshift-storage'
```

Comment:
Any reason we'd like to revert to openshift-storage? 
I think it'll be better to keep it until we fix the pod disappearance bug

Reply:
no, trying to change to rook-ceph